api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/PractitionerTranslatorUserImpl.java

```diff
@@ -54,7 +58,27 @@ public Practitioner toFhirResource(@Nonnull User user) {
 		practitioner.addIdentifier(userIdentifier);
 		
 		if (user.getPerson() != null) {
+			
+			if (user.getPerson().getBirthdateEstimated() != null) {
+				if (user.getPerson().getBirthdateEstimated()) {
+					DateType dateType = new DateType();
+					Calendar calendar = Calendar.getInstance();
```

Comment:
Instead of using `Calendar` can we change this to use the `java.time` package classes?

Reply:
changed the calendar to java.time package classes.